main.cpp

```diff
@@ -52,106 +52,115 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: lion
+//  action 1: the lion eats prey
+lion.eatPrey();
+//  action 2: the lion sleeps in a cave
+lion.sleepInCave();
+//  action 3: the lion feeds her children
+lion.feedChildren();
 
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: dolphin
+//  action 1: the dolphin jumps out of water
+lion.jumpOutWater();
+//  action 2: the dolphin fights a shark
+dolphin.fightShark();
+//  action 3: the dolphin becomes sentient
+dolphin.discoverReality();
 
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: Brooklyn
+//  action 1: Brooklyn snows
+brooklyn.heavySnow();
+//  action 2: Brooklyn opens a new park
+brooklyn.createPark();
+//  action 3: Brooklyn just hosted a marathon
+brooklyn.hostMarathon();
 
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: Michael Jordan
+//  action 1: Michael Jordan releases a new shoe
+mj.releaseNewShoe();
+//  action 2: MJ shoots three point attempt
+mj.shootThreePointer();
+//  action 3: MJ answers post game questions
+mj.answerReporter();
 
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: Spongebob
+//  action 1:  Spongebob makes a krabby patty
+  spongebob.cookKrabbyPatty();
+//  action 2: Spongebob feeds Gary
+  spongebob.feedGary();
+//  action 3: Spongebob tries to catch jellyfish
+  spongebob.jellyfishCatchAttempt();
 
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: maid
+//  action 1: a Maid dusts
+  maid.dust();
+//  action 2: a Maid sweeps the floor
+  maid.sweepFloor();
+//  action 3: a maid washes towels
+  maid.washTowel();
 
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: skateboard
+//  action 1: a skateboard breaks
+  skateboard.destroy();
+//  action 2: a skateboard get its wheels replaced
+  skateboard.changeWheels();
+//  action 3: a skateboard flips over
+  skateboard.flip();
 
 //  8)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: EQ
+//  action 1: an EQ is bypassed
+  EQ.bypass();
+//  action 2: an EQ is automated via touch
+  EQ.touchAutomation();
+//  action 3: an EQ is muted
+  EQ.mute();
 
 //  9)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: Reverb
+//  action 1: a reverb has its predelay modulated
+  reverb.modulatePreDelay();
+//  action 2: a reverb has a high pass filter added
+reverb.activateHighPassFilter();
+//  action 3: a reverb has its serial code activated for a paid subscription
+  rever.activateSubscriber();

```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.